packages/react-core/__tests__/operations/groupByDate.test.js

```diff
@@ -14,127 +14,141 @@ const DATES_VALUES = [
 
 const DATE_COLUMN = 'test_date';
 const OPERATION_COLUMN = 'test_revenue';
+const OPERATION_COLUMN_2 = 'test_revenue_2';
 
 const FEATURES = DATES_VALUES.map((value) => ({
   [DATE_COLUMN]: value,
-  [OPERATION_COLUMN]: REVENUE
+  [OPERATION_COLUMN]: REVENUE,
+  [OPERATION_COLUMN_2]: REVENUE
 }));
 
 describe('groupValuesByDateColumn', () => {
   test('should return null due to empty data array', () => {
-    expect(groupValuesByDateColumn([])).toEqual(null);
+    expect(groupValuesByDateColumn({ data: [] })).toEqual(null);
   });
 
   describe('valid features', () => {
     test('should return null due to invalid grouping operation', () => {
       expect(
-        groupValuesByDateColumn(
-          FEATURES,
-          OPERATION_COLUMN,
-          DATE_COLUMN,
-          '__fake_group_type__',
-          AggregationTypes.COUNT
-        )
+        groupValuesByDateColumn({
+          data: FEATURES,
```

Comment:
Having homogeneous naming for fields in group-functions looks good to me. Out of curiosity: why 'data' and not 'features'?

Reply:
I think that data was used in two fns and that's why I chose that, but features is also ok. What do you prefer?